packages/cli/src/ceramic-daemon.ts

```diff
@@ -344,24 +344,34 @@ export class CeramicDaemon {
 
     // Now that ceramic node is set up we can start publishing metrics
     // publishing metrics is enabled by default, even if no metrics config
-    if (! opts.metrics || opts.metrics?.metricsPublisherEnabled) {
+    if (!opts.metrics || opts.metrics?.metricsPublisherEnabled) {
       const ipfsVersion = await ipfs.version()
-      NodeMetrics.start({
-        ceramic: ceramic,
-        network: params.networkOptions.name,
-        ceramicVersion: version,
-        ipfsVersion: ipfsVersion.version,
-        intervalMS: opts.metrics?.metricsPublishIntervalMS || DEFAULT_PUBLISH_INTERVAL_MS,
-        nodeId: ipfsId.publicKey, // what makes the best ID for the node?
-        nodeName: '', // daemon.hostname is not useful
-        nodeAuthDID: did.id,
-        nodeIPAddr: '', // daemon.hostname is not the external name
-        nodePeerId: ipfsId.publicKey,
-        logger: diagnosticsLogger,
-      })
-      diagnosticsLogger.imp(
-        `Publishing Node Metrics publicly to the Ceramic Network.  To learn more, including how to disable publishing, please see the NODE_METRICS.md file for your branch, e.g. https://github.com/ceramicnetwork/js-ceramic/blob/develop/docs-dev/NODE_METRICS.md`
-      )
+      let didId: string | null = null;
```

Comment:
Seems like we can simplify this logic. Can we just check if `did.id` is set and then use that in the `if` check below?

Reply:
no, because the getter for did.id is what throws an error

Reply:
@smrz2001 this is where the error was being thrown, I believe we have to catch it: https://github.com/ceramicnetwork/js-did/blob/main/packages/dids/src/did.ts#L192

Reply:
Can we use the `authenticated` method instead?